chia/cmds/keys_funcs.py

```diff
@@ -178,8 +178,8 @@ def sign(message: str, private_key: PrivateKey, hd_path: str, as_bytes: bool):
 
 def verify(message: str, public_key: str, signature: str):
     messageBytes = bytes(message, "utf-8")
-    public_key = G1Element.from_bytes(bytes.fromhex(public_key))
-    signature = G2Element.from_bytes(bytes.fromhex(signature))
```

Comment:
how can you be sure that we can trust this key and signature?

Reply:
We now perform the subgroup checks inside the verify method